### PR TITLE
Fix for mythril action

### PR DESCRIPTION
**Pull request by iamsahu**
## Description

Mythril GitHub actions were not running. This fixes it

## Notice

- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/liquid-collective/liquid-collective-protocol/pulls) for the same update/change?
- [x] Have you assigned this PR to yourself?
- [x] Have you added at least 1 reviewer?
- [ ] Have you updated the official documentation?
- [ ] Have you added sufficient documentation in your code?
- [ ] Have you added relevant tests to the official test suite?

## Pull Request Type

- [ ] 💫 New Feature (Breaking Change)
- [ ] 💫 New Feature (Non-breaking Change)
- [ ] 🛠️ Bug fix (Non-breaking Change: Fixes an issue)
- [x] 🕹️ Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)

## Manual tests (if applicable)

The actions were checked manually

### CODE REVIEW

File: .github/workflows/Mythril.yaml [hunk 1]
```diff
@@ -33,4 +37,4 @@ jobs:
         uses: actions/upload-artifact@v3
         with:
           name: ${{ matrix.contract.name }}.mythril.md
-          path: ${{ matrix.contract.name }}.mythril.md
+          path: ${{ matrix.contract.name }}.mythril.md
```
Comment (cebidhem) [inline .github/workflows/Mythril.yaml:40]: @iamsahu can you please fix the EoF ?

**Review by cebidhem — CHANGES_REQUESTED** [review]

**Review by mischat — COMMENTED** [review]
is this now working? The PR is currently in draft, are you happy with it is @iamsahu ?

File: .github/workflows/Mythril.yaml [hunk 2]
```diff
@@ -11,11 +11,15 @@ jobs:
...
-          - { file: Withdraw.1.sol, name: WithdrawV1}
-          - { file: Allowlist.1.sol, name: AllowlistV1}
-          - { file: Firewall.sol, name: Firewall}
+          - { file: Withdraw.1.sol, name: WithdrawV1 }
+          - { file: Allowlist.1.sol, name: AllowlistV1 }
+          - { file: Firewall.sol, name: Firewall }
+          - { file: RedeemManager.1.sol, name: RedeemManagerV1 }
+          - { file: ELFeeRecipient.1.sol, name: ELFeeRecipientV1 }
+          - { file: OperatorsRegistry.1.sol, name: OperatorsRegistryV1 }
+          - { file: CoverageFund.1.sol, name: CoverageFundV1 }
```
Comment (mischat) [inline .github/workflows/Mythril.yaml:22]: do we want to add in TLC ? And is CoverageFund its own contract that is deployed, if it is why isn't it mentioned in the table on the README page ?

Comment (mischat) [reply .github/workflows/Mythril.yaml:22]: cc @iamsahu

Comment (iamsahu) [reply .github/workflows/Mythril.yaml:22]: I had missed TLC an have added it now

Comment (mischat) [reply .github/workflows/Mythril.yaml:22]: in which commit ? @iamsahu

Comment (iamsahu) [reply .github/workflows/Mythril.yaml:22]: @mischat : [This](https://github.com/liquid-collective/liquid-collective-protocol/pull/244/commits/8fee059ed1341ada065397d984664cef74387fbc) is the commit

Comment (mischat) [reply .github/workflows/Mythril.yaml:22]: great

**Review by mischat — DISMISSED** [review]
LGTM ! :)

**Review by mischat — APPROVED** [review]
:shipit:

**Review by cebidhem — APPROVED** [review]
LGTM 🚀

**Pull request merged**